tests/phpunit/tests/xmlrpc/wp/newPost.php

```diff
@@ -300,9 +346,13 @@ function test_terms_names() {
 	 */
 	function test_invalid_post_date_does_not_fatal() {
 		$this->make_user_by_role( 'author' );
-		$date_string = 'invalid_date';
-		$post = array( 'post_title' => 'test', 'post_content' => 'test', 'post_date' => $date_string );
-		$result = $this->myxmlrpcserver->wp_newPost( array( 1, 'author', 'author', $post ) );
+		$date_string  = 'invalid_date';
+		$post         = array(
+			'post_title'   => 'test',
+			'post_content' => 'test',
+			'post_date'    => $date_string,
+		);
+		$result       = $this->myxmlrpcserver->wp_newPost( array( 1, 'author', 'author', $post ) );
```

Comment:
Should items in this array start on new lines since it has more than one item?

Reply:
It's not an associative array.